runtime-feature-detection.md

```diff
@@ -0,0 +1,156 @@
+# Runtime Feature Detection on Graviton
+
+Different generations of Graviton support different features. For example, Graviton3 supports SVE but Graviton2 does
+not. Graviton4 supports SVE2 and SVE. For some applications it may be adventageous to implement performance critical
+kernels to make use of the highest performing feature set available which may not be known until runtime. For this, the
+best practice is to consult HWCAPS. This guide covers the procedure for feature detection on Linux.
+
+On Linux, the standard C library implements `getauxval` to read from the auxiliary vector. Reading the `AT_HWCAP` and
+`AT_HWCAP2` values will return the hardware capability bitmaps which are filtered by the kernel to include only features
```

Comment:
"filtered by the kernel" is very important, and kind of buried in this paragraph.